src/main/java/org/hpccsystems/jdbcdriver/HPCCConnection.java

```diff
@@ -40,19 +40,22 @@
 import java.util.concurrent.Executor;
 import java.util.logging.Level;
 
-import org.hpccsystems.ws.client.extended.HPCCWsSQLClient;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.ECLWorkunit;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.ExecutePreparedSQLResponse;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.ExecuteSQLResponse;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.GetResultsResponse;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.HPCCQuerySet;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.HPCCTable;
-import org.hpccsystems.ws.client.gen.extended.wssql.v3_05.NamedValue;
+import org.hpccsystems.ws.client.HPCCWsClient;
+import org.hpccsystems.ws.client.HPCCWsSQLClient;
+import org.hpccsystems.ws.client.gen.axis2.wssql.v3_05.NamedValue;
```

Comment:
It's too bad the executePreparedSQL call doesn't use the wrapper for NamedValue, maybe the thought was that the NamedValues structure won't change over time.
https://track.hpccsystems.com/browse/JAPI-378